wordpress-mobile/WordPress-iOS
# Comment Moderation: when Next is tapped on the snackbar, show the next comment.

Ref: #17290 

When `Next` is tapped on the comment moderation confirmation snackbar, the comment details view is updated with the next comment in the current filter. If the currently displayed comment is the last one, `Next` is not displayed on the snackbar.

To test:
- Enable the `newCommentDetail` feature.
- Go to My Site > Comments.
- Select a comment and moderate it. Stay on the details view.
  - Tap `Next` on the snackbar.
  - Verify the comment details view is updated with the next comment in the list.
- Select the last comment in a list and moderate it. Stay on the details view.
  - Verify the confirmation snackbar does not have a `Next` button.
 
NOTE: The comment content field height is not updated correctly when the next comment has less content than the prior one. The same problem occurs when editing a comment to have less content. As that is not related to this change, I'll leave that to be addressed later.

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is incomplete.

PR submission checklist:
